packages/@sanity/dashboard/src/widgets/projectInfo/ProjectInfo.js

```diff
@@ -147,54 +141,69 @@ class ProjectInfo extends React.PureComponent {
+              <Heading size={1} as="h2" id="project_info_table">
+                Project info
+              </Heading>
+            </Box>
             {this.assembleTableRows().map((item) => {
               if (!item || !item.rows) {
                 return null
               }
 
               return (
-                <tbody key={item.title}>
-                  <tr className={styles.sectionHeaderRow}>
-                    <th colSpan="2">{item.title}</th>
-                  </tr>
-                  {item.rows.map((row) => {
-                    return (
-                      <tr key={row.title}>
-                        <th className={styles.rowTitle}>{row.title}</th>
-                        {isPlainObject(row.value) && (
-                          <td className={styles.apiError}>{row.value.error}</td>
-                        )}
-                        {!isPlainObject(row.value) && (
-                          <td>
-                            {isUrl(row.value) ? <a href={row.value}>{row.value}</a> : row.value}
-                          </td>
-                        )}
-                      </tr>
-                    )
-                  })}
-                </tbody>
+                <Stack key={item.title} space={3}>
+                  <Card borderBottom padding={3}>
+                    <Label size={0} muted role="columnheader">
+                      {item.title}
+                    </Label>
+                  </Card>
+                  <Stack space={4} paddingX={3} role="rowgroup">
+                    {item.rows.map((row) => {
+                      return (
+                        <Grid key={row.title} columns={2} role="row">
+                          <Text weight="medium" role="rowheader">
+                            {row.title}
+                          </Text>
+                          {isPlainObject(row.value) && <Text size={1}>{row.value.error}</Text>}
+                          {!isPlainObject(row.value) && (
+                            <>
+                              {isUrl(row.value) ? (
+                                <Text size={1} role="cell" style={{wordBreak: 'break-word'}}>
+                                  <a href={row.value}>{row.value}</a>
+                                </Text>
+                              ) : (
+                                <Code size={1} role="cell" style={{wordBreak: 'break-word'}}>
+                                  {row.value}
+                                </Code>
+                              )}
+                            </>
+                          )}
+                        </Grid>
+                      )
+                    })}
+                  </Stack>
+                </Stack>
               )
             })}
-          </table>
-
-          <div className={styles.footer}>
-            <AnchorButton href={getManageUrl()} bleed color="primary" kind="simple">
-              Manage project
-            </AnchorButton>
```

Comment:
Hmm, why did we remove this link?

Reply:
Fixed, thank you 👍

Reply:
The URL doesn't actually work anymore, I realize. We should either grab the `organizationId` from the project info response above and build the correct URL (`/:orgId/project/:projectId`) or (as I suggested on Slack) build a route on manage that does the resolving and redirects